covid19-etl/etl/jhu_to_s3.py

```diff
@@ -163,12 +168,36 @@
         },
         ...
     }
+
+    (5) Choropleth IL map JSON by date:
+    Values are truncated or excluded to reduce file size
+    Date only added if C or D has changed from previous date
+    {
+        # IL only
+        "il_county_list": {
+            <US county FIPS>: {
+                "county": <county name>,
+                "by_date": {
+                    "<date>": {
+                        "C": <confirmed>
+                        "D": <deaths>
```

Comment:
could you explain what "C" and "D" mean in the comment instead of just "Values are truncated"